package.json

```diff
@@ -13,9 +13,10 @@
   ],
   "scripts": {
     "clear": "rimraf dist && mkdirp dist",
-    "build": "yarn clear && yarn svg && yarn build:prod",
-    "build:dev": "webpack --mode development --progress --display-error-details --display-entrypoints --watch",
-    "build:prod": "webpack --mode production",
+    "build": "yarn clear && yarn svg && yarn build:webpack:prod",
+    "build:watch": "yarn clear && yarn svg && yarn build:webpack:dev",
```

Comment:
why it was renamed?

Reply:
two main commands for complete bundle building and two commands for webpack run